### ISSUE TITLE
suggestion

### ISSUE THREAD

**Issue by ghost** [zertovitch/azip#2]
hi, I think it would be nice if the program had an option to "extract to" on right-click - similar to what 7zip and winrar have. currently, you have to open the program to be able to extract something, which is very inconvenient.

**Comment by zertovitch** [comment]
Agreed. Also, reversely, I miss the "Add folder to [folder].zip" one.
I have updated the to-do list with your suggestion: https://github.com/zertovitch/azip/commit/cc460b41dec5c966b34a847d41aceb4ac91f190a.

**Comment by ghost** [comment]
Nice. Once that's done, Azip will become much more convenient to use. You could also add other functions that are present on the gui, such as Test Archive, Recompress, Add file with encryption, and so on.

**Comment by ghost** [comment]
also, and I'm not sure if this is intentional or not, but if you click on "new archive" when an archive is already open, it will basically stay on top of the other ones, requiring you to click on an awkwardly-placed window bar to close the new archive and return to the old one(s). my suggestion is to either create a drop down archive selector, or just ditch it and use new windows for new archives.
